src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -45,13 +45,13 @@ public AbstractFindCommand parse(String args) throws ParseException {
 
         // return appropriate FindCommand class depending on tag
         switch (tag) {
-        case "/n":
+        case "n/":
             return new FindByNameCommand(
                     new NameContainsKeywordsPredicate(Arrays.asList(searchTermArray)));
-        case "/c":
+        case "c/":
             return new FindByContactCommand(
                     new ContactContainsKeywordsPredicate(Arrays.asList(searchTermArray)));
-        case "/e":
+        case "e/":
             return new FindByEmailCommand(
                     new EmailContainsKeywordsPredicate(Arrays.asList(searchTermArray)));
         default:
```

Comment:
Can keep in view to shorten the code here by establishing Arrays.asList(....) outside the switch so we do not have to type this 3 times